lib/runtime/wasmer/exports_test.go

```diff
@@ -408,6 +408,96 @@ func TestInstance_GrandpaAuthorities_PolkadotRuntime(t *testing.T) {
 	require.Equal(t, expected, auths)
 }
 
+func TestInstance_BabeGenerateKeyOwnershipProof_NodeRuntime(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+
+	rvalue, err := common.HexToHash("0x01")
+	require.NoError(t, err)
+	key := common.MustHexToBytes(genesis.BABERandomnessKeyHex)
+	tt.Put(key, rvalue[:])
+
+	avalue, err := common.HexToBytes("0x08eea1eabcac7d2c8a6459b7322cf997874482bfc3d2ec7a80888a3a7d714103640100000000000000b64994460e59b30364cad3c92e3df6052f9b0ebbb8f88460c194dc5794d6d7170100000000000000") //nolint:lll
+	require.NoError(t, err)
+
+	key = common.MustHexToBytes(genesis.BABEAuthoritiesKeyHex)
+	tt.Put(key, avalue)
+
+	rt := NewTestInstanceWithTrie(t, runtime.NODE_RUNTIME, tt)
+
+	babeConfig, err := rt.BabeConfiguration()
+	require.NoError(t, err)
+
+	require.NotEmpty(t, babeConfig.GenesisAuthorities)
+
+	authorityID := babeConfig.GenesisAuthorities[0].Key
+
+	const slot = uint64(10)
+	_, err = rt.BabeGenerateKeyOwnershipProof(slot, authorityID)
+	require.NoError(t, err)
+}
+
+func TestInstance_BabeGenerateKeyOwnershipProof_PolkadotRuntime(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+
+	rvalue, err := common.HexToHash("0x01")
+	require.NoError(t, err)
+	key := common.MustHexToBytes(genesis.BABERandomnessKeyHex)
+	tt.Put(key, rvalue[:])
+
+	avalue, err := common.HexToBytes("0x08eea1eabcac7d2c8a6459b7322cf997874482bfc3d2ec7a80888a3a7d714103640100000000000000b64994460e59b30364cad3c92e3df6052f9b0ebbb8f88460c194dc5794d6d7170100000000000000") //nolint:lll
+	require.NoError(t, err)
+
+	key = common.MustHexToBytes(genesis.BABEAuthoritiesKeyHex)
+	tt.Put(key, avalue)
+
+	rt := NewTestInstanceWithTrie(t, runtime.POLKADOT_RUNTIME, tt)
```

Comment:
The `POLKADOT_RUNTIME` targets to polkadot 0.8.25 (old one) could you change to `POLKADOT_RUNTIME_v0929`?